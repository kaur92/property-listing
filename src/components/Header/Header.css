.header {
  align-items: center;
  display: flex;
  padding: 10px 10px;
  box-shadow: 0px 0px 21px 0px rgba(200, 200, 200, 1);
}

.logo {
  width: 160px;
}
