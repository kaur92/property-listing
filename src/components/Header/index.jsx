import React from "react";
import logo from "./logo.svg";
import "./Header.css";

const Header = () => (
  <header className="header">
    <img src={logo} alt="" className="logo" />
  </header>
);

export default Header;
