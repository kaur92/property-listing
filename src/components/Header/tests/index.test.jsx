import renderer from "react-test-renderer";
import Header from "../index";

describe("<Header /> tests", () => {
  it("renders correctly", () => {
    const cardComponent = renderer.create(<Header />).toJSON();
    expect(cardComponent).toMatchSnapshot();
  });
});
