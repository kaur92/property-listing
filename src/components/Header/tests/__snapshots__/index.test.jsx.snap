// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> tests renders correctly 1`] = `
<header
  className="header"
>
  <img
    alt=""
    className="logo"
    src="logo.svg"
  />
</header>
`;
